src/pages/Main/Banners/BannerList.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import styled, { keyframes, css } from 'styled-components';
+
+const BannerList = props => {
+  const { image, title, subtitle, background, idx } = props.banner;
+  const { currentSlide } = props;
+
+  return (
+    <List background={background} currentSlide={currentSlide} idx={idx}>
+      <Container>
+        <Title>{title}</Title>
+        <SubTitle>{subtitle}</SubTitle>
+        <Img src={image} />
+        <Button direction={'left'} onClick={e => props.handleSlide(e, 'left')}>
+          <i className="fas fa-chevron-left" />
+        </Button>
+        <Button onClick={e => props.handleSlide(e, 'right')}>
+          <i className="fas fa-chevron-right" />
+        </Button>
+        <IndexButton>
+          {idx} / 5
+          <i className="fas fa-chevron-right" />
+        </IndexButton>
+      </Container>
+    </List>
+  );
+};
+
+const List = styled.li`
+  position: absolute;
+  width: 100%;
+  height: 400px;
+  padding: 40px 0 0 0;
+  background: ${props => props.background};
+  opacity: ${props => (props.currentSlide === props.idx ? 1 : 0)};
+`;
+
+const Container = styled.div`
+  position: relative;
+  max-width: 900px;
+  height: 100%;
+  padding: 0 60px;
+  margin: 0 auto;
+`;
+
+const Title = styled.h1`
+  margin-bottom: 12px;
+  font-size: 2rem;
+  font-weight: bold;
+`;
+
+const SubTitle = styled.h2`
+  color: ${props => props.theme.deepGrey};
+  font-size: 1.1rem;
+`;
+
+const Img = styled.img`
+  position: absolute;
+  bottom: -20px;
+  right: 80px;
+  width: 150px;
+  height: auto;
+`;
+
+const Button = styled.button`
+  position: absolute;
+  top: 50%;
+  ${props => (props.direction === 'left' ? 'left:0' : 'right:0')};
+  font-size: 1.5rem;
+  transform: translateY(-50%);
+`;
+
+const IndexButton = styled.div`
+  position: absolute;
+  bottom: 20px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 80px;
+  height: 30px;
+  border: 1px solid ${props => props.theme.black};
```

Comment:
```suggestion
  border: 1px solid ${({ theme }) => theme.black};
```
여기도 구조분해 할당 가능해보입니다!

Reply:
몰랐었는데 감사합니다! 
구조분해 할당 더 잘 사용해보도록 하겠습니다!